Makefile

```diff
@@ -0,0 +1,102 @@
+# Makefile
+# Shortcuts to invoke OSISM testbed stack generation
+# Pass STACKNAME=XXX to change the name of the deployed stack (default: testbed),
+# STACK_PARAMS= (e.g. --parameter XYZ=abs) if you want to pass parameter to heat
+# TMPL_PARAMS= (e.g. -Dnumber_of_volumes=4) if you want to modify the stack
+#  generation from the template.
+# 
+# (c) Kurt Garloff <scs@garloff.de>, 3/2020, Apache2 License
+
+STACKNAME = testbed
+
+default: stack.yml stack-single.yml
+
+stack.yml: templates/stack.yml.j2
+	jinja2 -o $@ $(TMPL_PARAMS) $^
+
+stack-single.yml: templates/stack.yml.j2
+	jinja2 -o $@ $(TMPL_PARAMS) -Dnumber_of_nodes=0 $^
+
+dry-run: stack.yml environment.yml
+	openstack stack create --dry-run -t $< -e environment.yml $(STACK_PARAMS) $(STACKNAME) -f json; echo
+
+deploy: stack.yml environment.yml
```

Comment:
That's unclear. I would use create here instead of deploy.

Reply:
If we have targets like deploy-infra, deploy-ceph, deploy-openstack, isn't a nakes deploy the most logical. Happy to create a "create" alias, though, if that works for you.